src/main/java/uk/gov/hmcts/reform/pip/data/management/service/PublicationService.java

```diff
@@ -360,10 +377,26 @@ public void reportNoMatchArtefacts() {
     }
 
     /**
-     * Delete expired artefacts from database and Azure storage.
+     * Delete expired artefacts from the database, Artefact and Publications Azure storage.
      */
     public void deleteExpiredArtefacts() {
-        deleteExpiredBlobs(artefactRepository.findOutdatedArtefacts(LocalDate.now()));
```

Comment:
This logic also needs to be done in the deleteArtefactById method too

Reply:
Done